src/test/java/uk/gov/justice/laa/crime/cfecrime/utils/RequestTestUtil.java

```diff
@@ -27,4 +40,61 @@ public static void setSectionPassportBenefit(CfeCrimeRequest request, boolean va
         sectionPassportedBenefit.withPassportedBenefit(Boolean.valueOf(value));
         request.withSectionPassportedBenefit(sectionPassportedBenefit);
     }
+
+    public static void setSectionInitMeansTest(CfeCrimeRequest cfeCrimeRequest, CaseType caseType, MagCourtOutcome magCourtOutcome)  {
+        List<DependantChild> dcList = new ArrayList<DependantChild>();
+        DependantChild dc = new DependantChild();
+        dc.setAgeRange(AgeRange.SIXTEEN_TO_EIGHTEEN);
+        dcList.add(dc);
```

Comment:
This feels strange. (a) its arbitrary data (b) by using the 'setter' pattern we have set 0 children in the bucket (or null even!) which isn't valid

Reply:
It is test data, we can work out what needs to be added/removed.